### PR TITLE
Isan/home uiux impl

### LINKED ISSUE #41: Create List Page using UI/UX design implementation
image

Check on figma for details

### PR DESCRIPTION
**Pull request by ichsanachmad**
close #41

### CODE REVIEW

File: lib/views/screens/home_screen/pages.dart [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+          ),
+        ),
+        Container(
+          height: 130,
+          margin: EdgeInsets.only(top: 12),
+          // Change real data from API with ListView.builder()
+          child: ListView(
+            scrollDirection: Axis.horizontal,
+            children: [
+              Utils.productCategoryListItem(
```
Comment (ariefwijaya) [inline lib/views/screens/home_screen/pages.dart]: Jangan return pake `Utils.productCategoryListItem` gini san, ini bad practices, bakal reduce performance aplikasi. makin banyak card yg dipakek makin signifikan turunnyo. ngecek benchmarkny via dart dev tools ketika debug.

Solusiny, buat nian class cardnyo pakek statelesswidget,

**Review by ariefwijaya — CHANGES_REQUESTED** [review]
Please Revise.

File: lib/views/screens/home_screen/pages.dart (hunk 1 above)
Comment (ichsanachmad) [reply lib/views/screens/home_screen/pages.dart]: oke sip kak

**Review by ariefwijaya — APPROVED** [review]
LGTM. nice!

**Pull request merged**